server/backend/test/steps/get-notifications.steps.ts

```diff
@@ -0,0 +1,180 @@
+import { INestApplication } from '@nestjs/common';
+import { defineFeature, loadFeature } from 'jest-cucumber';
+import request from 'supertest';
+import { Camera } from '../../src/cameras/camera.entity';
+import { Notification } from '../../src/cameras/notification.entity';
+import {
+  buildTestStack,
+  TestStack,
+  TEST_STACK_TIMEOUT,
+} from '../helpers/test-stack';
+import { EntityManager, EntityRepository } from '@mikro-orm/core';
+import { MikroORM } from '@mikro-orm/core';
+import { CamerasModule } from '../../src/cameras/cameras.module';
+import { Group } from '../../src/groups/group.entity';
+import { User } from '../../src/users/user.entity';
+import { createCameraWithKeyPair, getCameraAuthToken } from '../helpers/camera';
+
+const feature = loadFeature('test/features/get-notifications.feature');
+
+defineFeature(feature, (test) => {
+  let app: INestApplication;
+  let testStack: TestStack;
+  let cameraRepository: EntityRepository<Camera>;
+  let em: EntityManager;
+
+  beforeAll(async () => {
+    testStack = await buildTestStack({ imports: [CamerasModule] });
+
+    em = testStack.module.get<MikroORM>(MikroORM).em.fork();
+    cameraRepository = em.getRepository(Camera);
+
+    app = testStack.module.createNestApplication();
+    await app.init();
+  }, TEST_STACK_TIMEOUT);
+
+  afterAll(async () => {
+    await app.close();
+    await testStack.dbContainer.stop();
```

Comment:
nit: We also need to remember to stop the Keycloak container after our tests (I think testcontainers might do this already, but we should stop it just to be safe). I'll refactor `TestStack` later so we don't have to do this.
```suggestion
    await app.close();
    await testStack.kcContainer.stop();
    await testStack.dbContainer.stop();
```